### PR TITLE
enable Copy and Select All

### PR DESCRIPTION
**Pull request by abukin**
Related to Issue #51  - I enabled Select All and Copy items in the Edit menu, along with corresponding keyboard shortcuts.
I tested with OS X 10.11 and Ubuntu 15.04. I don't have a windows box to test with, but should work there as well.

### CODE REVIEW

**Comment by maxkueng** [comment]
Nice! It works when I copy or select all using the keyboard shortcuts. But when I try it using the "Edit" menu I get the following error:

error message

According to the [docs](https://github.com/atom/electron/blob/master/docs/api/menu.md#menusendactiontofirstresponderaction-os-x), `Menu.sendActionToFirstResponder(action)` is an OSX-only feature. Could be nice if it worked on all supported platforms.

**Comment by abukin** [comment]
Yep it never occurred to me to check clicking on the actual menu items as opposed to using keyboard shortcuts :)

I changed it to use [`document.execCommand`](https://developer.mozilla.org/en-US/docs/Web/API/Document/execCommand) callback instead. Seems to work

File: client.js [hunk 1]
```diff
@@ -53,6 +53,13 @@ var template = [
     ]
   },
   {
+    label: 'Edit',
+    submenu: [
+        { label: 'Copy', accelerator: 'CmdOrCtrl+C', click: function() { document.execCommand("copy") } },
```
Comment (maxkueng) [inline client.js]:
```  
client.js:58:68: Missing space before function parentheses.
client.js:58:94: Strings must use singlequote.
```

File: client.js [hunk 2]
```diff
@@ -53,6 +53,13 @@ var template = [
...
+        { label: 'Select All', accelerator: 'CmdOrCtrl+A', click: function() { document.execCommand("selectAll") } }
```
Comment (maxkueng) [inline client.js]:
```
client.js:59:74: Missing space before function parentheses.
client.js:59:100: Strings must use singlequote.
```

**Comment by maxkueng** [comment]
Cool! This works! 
Can you make sure the tests pass? The only thing that's tested is whether the code validates against the "standard" code style thing. I've added notes to the respective lines.

After that we can merge it and I'm sure everybody will love this awesome feature.

**Comment by abukin** [comment]
sorry about that, I should've paid closer attention to the code style. lesson learned :)

**Comment by maxkueng** [comment]
:shipit: vmd@1.14.0 is on npm. Thanks!

**Pull request merged**